README.md

```diff
@@ -9,31 +9,63 @@ view-git-file macro on any Confluence page.
 ## Marketplace
 
 You can install Git for Confluence by using the Universal Plugin Manager inside
-Confluence, or by visiting the [Git for Confluence marketplace entry][1]
+Confluence, or by visiting the
+[Git for Confluence marketplace entry](https://avisi-apps.gitbook.io/git-for-confluence/.md).
 
 ## Documentation
 
 If you want to learn how to configure Git for Confluence or use the Git for
-Confluence macro's, please refer tot the [Git for Confluence documentation][2]
+Confluence macro's, please refer tot the
+[Git for Confluence documentation](https://avisi-apps.gitbook.io/git-for-confluence/).
```

Comment:
Idem

Reply:
Same here. Moved the link from the bottom to use it directly here.